packages/framework/request-handler/src/test/requestHandlers.spec.ts

```diff
@@ -63,43 +62,43 @@ async function assertRejected(p: Promise<IResponse | undefined>) {
 }
 
 describe("RequestParser", () => {
```

Comment:
This section `"RequestParser"` should just be removed.  With the change it would be duplicative of `"rootDataStoreRequestHandler"` below.